packages/esm-active-visits-app/src/routes.json

```diff
@@ -14,6 +14,16 @@
       "name": "visit-summary-widget",
       "slot": "visit-summary-slot",
       "component": "visitDetail"
+    },
+    {
+      "name": "home-active-visits-tile",
```

Comment:
Better for extension names to be fairly generic even if it default plugs into a slot, i.e., "active-visits-tile"